DesignHashMap.java

```diff
@@ -0,0 +1,65 @@
+//problem 2: 
+// Time Complexity : Amortized O(1)
+// Space Complexity : O(1)
+// Did this code successfully run on Leetcode : No
+// Any problem you faced while coding this : Only 26 test case passed but not sure why this logic is failing
+
+class MyHashMap {
+
+    private int[][] matrix;
+    int keys;
+    int keyItems;
+
+    private int hash1(int val) { // First hash function that do the modulo
+        return val % keys;
+    }
+
+    private int hash2(int val) {// First hash function that take do the division
+        return val / keyItems;
+    }
+
+    public MyHashMap() { // declare the variables
+        keys = 1000;
+        keyItems = 1000;
```

Comment:
keyItems = 1001 since the max key is 1_000_000, the hash2(1_000_000) will be 1000, so we need an array of size 1001 (0 to 1000 inclusive)